xsd/netex_framework/netex_genericFramework/netex_place_version.xsd

```diff
@@ -2,6 +2,7 @@
 <xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.1" id="netex_place_version">
 	<xsd:include schemaLocation="netex_place_support.xsd"/>
 	<xsd:include schemaLocation="../netex_genericFramework/netex_zoneProjection_version.xsd"/>
+	<xsd:include schemaLocation="../netex_reusableComponents/netex_mode_support.xsd"/>
```

Comment:
@ue71603 @Aurige when I remove this include, processing still works. The AccessModeEnumeration is only mentioned in netex_mode_support, so I am technically puzzled how it "correctly" ends up there.

Reply:
I checked here: it does not validate as a stand-alone file if you remove this include, it works only when included by others who have already included netex_mode_support.xsd ... so it is best keeping the include here